improver/synthetic_data/set_up_test_cubes.py

```diff
@@ -277,32 +368,9 @@ def set_up_variable_cube(
         domain_corner=domain_corner,
     )
 
-    # construct realization dimension for 3D data, and dim_coords list
-    ndims = len(data.shape)
-    if ndims == 3:
-        if realizations is not None:
-            if len(realizations) != data.shape[0]:
-                raise ValueError(
-                    "Cannot generate {} realizations from data of shape "
-                    "{}".format(len(realizations), data.shape)
-                )
-            realizations = np.array(realizations)
-            if issubclass(realizations.dtype.type, np.integer):
-                # expect integer realizations
-                realizations = realizations.astype(np.int32)
-            else:
-                # option needed for percentile & probability cube setup
-                realizations = realizations.astype(np.float32)
-        else:
-            realizations = np.arange(data.shape[0]).astype(np.int32)
-        realization_coord = DimCoord(realizations, "realization", units="1")
-        dim_coords = [(realization_coord, 0), (y_coord, 1), (x_coord, 2)]
-    elif ndims == 2:
-        dim_coords = [(y_coord, 0), (x_coord, 1)]
-    else:
-        raise ValueError(
-            "Expected 2 or 3 dimensions on input data: got {}".format(ndims)
-        )
+    dim_coords = _get_dimension_coords(
```

Comment:
Would be good to match the naming convention elsewhere in the module - maybe `_construct_dimension_coords`?

Reply:
Updated [here](https://github.com/metoppv/improver/pull/1309/commits/1d7f21003ca70fbe79b8c8943e280bc37f411b98#diff-ea38258437543d699495d248fce61e29L246).